app/Http/Services/NcpoService.php

```
@@ -0,0 +1,131 @@
+    public function getSupplierDetails($purchaseOrder, $latestChange = null): array
+    {
+        $original = [
+            'id' => $purchaseOrder->supplier_id,
+            'name' => $purchaseOrder->supplier?->company_name,
+            'address' => $purchaseOrder->supplier?->company_address,
+            'contact_number' => $purchaseOrder->supplier?->company_contact_number,
+        ];
+        if (!$latestChange || !$latestChange->changed_supplier_id) {
+            return $original;
+        }
+        $changedSupplier = RequestSupplier::find($latestChange->changed_supplier_id);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\RequestSupplier' in method 'getSupplierDetails'. (undefined)

(StaticAccess)